definitions/v7/lat-team-api.yml

```diff
@@ -97,8 +93,6 @@ search:
   keywordsfilters:
     - name: re_replace
       args: ["\\.", " "]
-    - name: re_replace
-      args: ["(?i)\\bS(\\d+)", "T$1"]
```

Comment:
this fixes TXXEYY being added as a search string as discussed on discord.

However as line 90 still exists, every query will have `&name=SXXEYY` append to it.

Likely the name input should be moved as part of the raw string conditional string.

alternatively, if the site works and has no issues with name appended then this is fine.

Reply:
I just tried with complete shows and single episodes and this new .yml works without issue. Line 90 is exactly the same as in blutopia's .yml.